tests/console/ansible.pm

```diff
@@ -0,0 +1,107 @@
+# SUSE's openQA tests
+#
+# Copyright 2022 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+# Package: ansible
+# Summary: Test ansible
+# Maintainer: Pavel Dostál <pdostal@suse.cz>
+
+use warnings;
+use base consoletest;
+use strict;
+use testapi qw(is_serial_terminal :DEFAULT);
+use utils qw(zypper_call clear_console systemctl);
+
+sub run {
+    my $self = shift;
+    $self->select_serial_terminal;
+
+    # Instal Ansible from PIPY
+    assert_script_run 'pip3 install ansible', 300;
+
+    # Enable sshd
+    systemctl 'start sshd';
+
+    # add $testapi::username to sudoers without password
+    assert_script_run "echo '$testapi::username ALL=(ALL:ALL) NOPASSWD: ALL' | tee -a /etc/sudoers";
```

Comment:
Please create the file in `sudoers.d` instead, as it would help with the cleanup step

Reply:
That is cool suggestion